docs/topics/multiplatform/multiplatform-compatibility-guide.md

```diff
@@ -910,6 +912,41 @@ Here's the planned deprecation cycle:
   with "unresolved reference" errors, and binaries (for example, Gradle plugins) might fail with linkage errors
   unless recompiled against the latest versions of the Kotlin Gradle plugin
 
+<anchor name="target-shortcuts-deprecation"/>
+## Deprecated Apple target shortcuts
+
+**What's changed?**
+
+We're deprecating `ios()`, `watchos()`, and `tvos()` target shortcuts in Kotlin Multiplatform DSL. They were designed to
+partially create a source set hierarchy for Apple targets. However, they proved to be difficult to expand and sometimes confusing.
+
+For example, the `ios()` shortcut created both the `iosArm64` and `iosX64` targets but didn't include the `iosSimulatorArm64`
+target, which is necessary when working on hosts with Apple M chips. However, changing this shortcut was hard to implement
+and could cause issues in existing user projects.
+
+**What's the best practice now?**
+
+The Kotlin Gradle plugin now provides a built-in hierarchy template. Since Kotlin 1.9.20, it's enabled by default
+and contains predefined intermediate source sets for popular use cases.
+
+The difference from shortcuts is that you need to explicitly specify the set of targets first, and the plugin then
```

Comment:
```suggestion
Instead of shortcuts you should specify the list of targets, and then the plugin
```

"Explicitly specify" reads redundant to me here, but maybe it isn't?
Also, changed "set" to "list" so that it corresponds to "this list" later.